### DISCUSSION TITLE
Where is the source for the reference documentation?

### DISCUSSION THREAD

**Discussion by jdm** [oxidecomputer/hubris#296] [Q&A] [answered]
I found what looks like a mistake in it, and I was hoping to make a PR. I can't find it in https://github.com/oxidecomputer/hubris/tree/master/doc or in a separate repository under the oxidecommputer org, however.

**Comment by jdm** [comment, marked as answer]
Oops, I discovered that it's actually in the docs/ subdir of the hubris repository.

**Comment by stappersg** [comment]
On Sat, Dec 04, 2021 at 08:40:21PM -0800, Josh Matthews wrote:
> Oops, I discovered that it's actually in the docs/ subdir of the hubris repository.

Thanks for merge request #297


My request:   Close this discussion.

Why: To Prevent that this discussion drains further human energy.




In case github.com does assume

_Each opened discussion is worth to remain open forever._



then we can take comfort with

_This discussion is for documenting that it is good
to think twice and to think triple
before bothering way more people as you invite for a birthday party._

**Reply by cbiffle** [reply]
No need to be rude.
